game.js

```diff
@@ -0,0 +1,48 @@
+const COMP_WINS = 0;
+const USER_WINS = 1;
+
+const computerSelection = computerPlay();
+console.log(computerSelection);
+const buttons = document.querySelector(".js-buttons-container"); //selected parent container
+buttons.addEventListener("click", choosePlayer); // adds listener on each clicked button
+
+function choosePlayer(e) {
+  const playerSelection = e.target.value;
+  console.log(playerSelection);
+  e.stopPropagation(); //Stop the event propagation from running up and down the DOM
+  playRound(playerSelection, computerSelection);
```

Comment:
I suggest to display the result on the page instead of `console.log()`. People don't usually have devtools open when they browse Internet.